### PR TITLE
electrode-react-webapp changes in generator

### PR DESCRIPTION
**Pull request by donsalari**
@jchip @ananavati @animesh10 @jmcriffey

### CODE REVIEW

**Comment by jchip** [comment]
master is now updated, could you rebase and PR to that please?

File: .gitignore [hunk 1]
```diff
@@ -502,7 +502,6 @@ sauce_connect*.txt
 *.dmp
 
 ### Walmart Projects ###
-lib
```
Comment (donsalari) [inline .gitignore]: bc of this we ignored changes inside `packages/electrode-react-webapps/lib`

**Comment by donsalari** [comment]
travis build will always fail as we need to merge this PR and then publish `electrode-react-webapp` from `packages`

**Comment by jchip** [comment]
please PR into master branch

**Pull request closed without merging**